core-build/gulp-core-build/src/logging.ts

```diff
@@ -174,7 +174,6 @@ function afterStreamsFlushed(callback: () => void): void {
 }
 
 function writeSummary(callback: () => void): void {
-  const shouldRelogIssues: boolean = getFlagValue('relogIssues');
```

Comment:
>getFlagValue [](start = 37, length = 12)

why are we looking at getFlagValue. Shouldn't it be getConfigValue ?

Reply:
getFlagValue looks at the command line, the environment, and at the build config.